src/data/zwave_js.ts

```diff
@@ -628,6 +665,74 @@ export const subscribeZwaveNodeStatistics = (
     }
   );
 
+export const fetchZwaveNodeFirmwareUpdateProgress = (
+  hass: HomeAssistant,
+  device_id: string
+): Promise<boolean> =>
```

Comment:
I would expect something like `ZWaveJSNodeFirmwareUpdateProgressMessage` instead of a Boolean?

Reply:
Or the function name should be `fetchZwaveNodeIsFirmwareUpdateInProgress`